resources/appimage/AppDir/bin/cryptomator.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+cd $(dirname $0)
+
+BUILD_NUMBER=$(cat ./build.number)
+
+# determine GTK version
+GTK2_PRESENT=$(test -z "x") #dummy value
+GTK3_PRESENT=$(test -z "") #dummy value
+if [ -n $(which dpkg) ]; then #do stuff for debian based things
+	GTK2_PRESENT=$( test -z `dpkg -l libgtk* | grep -e '\^ii' | grep -e 'libgtk2-*'` )
+	GTK3_PRESENT=$( test -z `dpkg -l libgtk* | grep -e '\^ii' | grep -e 'libgtk3-*'` )
+elif [ -n $(which rpm) ]; then # do stuff for rpm based things
+	GTK2_PRESENT=$( test -z `rpm -qa | grep -e '\^gtk2-[0-9][0-9]*'` )
+	GTK3_PRESENT=$( test -z `rpm -qa | grep -e '\^gtk3-[0-9][0-9]*'` )
+elif [ -n $(which pacman)]; then #don't forget arch
+	pacman -Qi gtk2
+	GTK2_PRESENT=$( test $? -eq 0 )
+	pacman -Qi gtk3
+	GTK3_PRESENT=$( test $? -eq 0 )
+fi
+
+
+if $GTK2_PRESENT && (! $GTK3_PRESENT ); then
+	GTK_FLAG="-Djdk.gtk.version=2"
+else
+	GTK_FLAG="-Djdk.gtk.version=3"
```

Comment:
Is it correct to set this to `3`? Or should the complete line just be absent?

Reply:
And, if this is correct, maybe it would be nicer to store the version in a variable and then later have `-Djdk.gtk.version=${GTK_VERSION}` as a param in in `java` command.

Reply:
Is absent a viable option? I don't want the shell script choke on an empty variable. 

To answer your question: Basically absence is preferable, because then we just use the javafx default (currently GTK+3)

Reply:
Resolved by 4a88dd9